eth/api.go

```diff
@@ -257,6 +257,16 @@ func (api *PrivateAdminAPI) ImportChain(file string) (bool, error) {
 	return true, nil
 }
 
+// NewHead requests the node to beacon-sync to the designated head header.
+func (api *PrivateAdminAPI) NewHead(blob hexutil.Bytes) error {
+	header := new(types.Header)
+	if err := rlp.DecodeBytes(blob, header); err != nil {
+		return err
+	}
+	mode, _ := api.eth.handler.chainSync.modeAndLocalHead()
+	return api.eth.Downloader().BeaconSync(mode, header)
+}
+
```

Comment:
Is this a debug that slipped in? Because IIUC, calling `admin_NewHead` with a blob will trigger a beaconsync? Seems like a way to screw up your chain. Do we really want this in prod?

Reply:
I used this to test beacon syncing on the current network without the beacon sync APIs. It can be deleted.